packages/cinterop/src/darwin/kotlin/io/realm/internal/interop/RealmInterop.kt

```diff
@@ -1440,6 +1436,20 @@ actual object RealmInterop {
         realm_wrapper.realm_clear_cached_apps()
     }
 
+    actual fun realm_app_sync_client_get_default_file_path_for_realm(
+        app: RealmAppPointer,
+        syncConfig: RealmSyncConfigurationPointer,
+        name: String?
+    ): String {
+        val cPath = realm_wrapper.realm_app_sync_client_get_default_file_path_for_realm(
+            app.cptr(),
+            syncConfig.cptr(),
+            name
+        )
+        return cPath.safeKString()
+            .also { realm_wrapper.realm_free(cPath) }
```

Comment:
Not that I'm aware of. I did it because the C-API docs explicitly state the following:
```
/**
 * Get the default realm file path based on the user and partition value in the config.
 *
 * @param custom_filename custom name for the realm file itself. Can be null,
 *                        in which case a default name based on the config will be used.
 *
 * Return value must be manually released with realm_free().
 */
```
There are other C-API functions that require it but we haven't implemented them yet.

Reply:
:+1: Sounds like we need to go through all of them then 🙈